src/wmnds/assets/sass/_vars.scss

```diff
@@ -23,6 +23,7 @@ $palettes: (
 
 $white: #ffffff;
 $black: #000000;
+$dark-grey: #2b2b2b;
```

Comment:
Move the dark-grey variable into the _component-example.scss as it won't be used anywhere else.
Not only that but it will only be used on the design system website. The components never reference it.

Reply:
@catiacosta-wmca let me know when you've done this

Reply:
I would sware SCSS lint was complaining, but maybe it was looking to a previous error. It's updated now.